fix: first sentence in step 32 JS course.

WalidMoua opened the pull request:
Checklist:

<!-- Please follow this checklist and put an x in each of the boxes, like this: [x]. It will ensure that our team takes your pull request seriously. -->

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

<!--If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.-->

Closes #48787

<!-- Feel free to add any additional description of changes below this line -->
rewrote the first sentence in step 32 of the new JS course build an rpg, to improve the Arabic translation

WalidMoua committed:
fix: first sentence in step 32 JS course.

raisedadead commented:
We are officially on Node.js 18 now. We have changed the merge requirements accordingly.

You will need to rebase this PR against the main branch OR Click the "Update branch" button to get the new checks going.

Thanks for your contribution.

WalidMoua committed:
Merge branch 'freeCodeCamp:main' into patch-1

WalidMoua commented:
> We are officially on Node.js 18 now. We have changed the merge requirements accordingly.
> 
> You will need to rebase this PR against the main branch OR Click the "Update branch" button to get the new checks going.
> 
> Thanks for your contribution.

Thanks for the help,  I have rebased it

Dario-DC commented:
I added a comment in the issue that I think should be considered before reviewing this PR

Sboonny commented on curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62fc20387ef88d1d1998aac5.md:
```diff
@@ -7,7 +7,7 @@ dashedName: step-32
 # --description--
 
-Use `querySelector()` to get the other two button elements, using their `#button2` and `#button3` `id`s. Store them in variables called `button2` and `button3`. Remember to use `const`.
+Use `querySelector()` to get the other two `button` elements, using their `id`s:  `#button2` and `#button3`  Store them in variables called `button2` and `button3`. Remember to use `const`.
```
Here is the new suggested string mention in the comment

```suggestion
Use `querySelector()` to get the other two `button` elements using their `id`s: `button2` and `button3`.
```

WalidMoua committed:
Update curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62fc20387ef88d1d1998aac5.md

Co-authored-by: Muhammed Mustafa <muhammed@freecodecamp.org>

WalidMoua commented:
The PR have been updated according to the new suggestion

Sboonny reviewed (approved):
#### :sparkles: :tada: **AWESOME!** :tada: :sparkles:

 Hi @WalidMoua, thanks for this pull request and for contributing to the code-base for the first time. We are looking forward to more contributions from you in the future.

 Cheers & happy contributing!

WalidMoua committed:
Merge branch 'main' into patch-1
